packages/replay/src/types.ts

```diff
@@ -285,6 +285,7 @@ export interface ReplayPluginOptions extends ReplayNetworkOptions {
       scrollTimeout: number;
       ignoreSelectors: string[];
     };
+    delayFlushOnCheckout: number;
```

Comment:
m: As long as we keep this an experiment, I'm fine with this name but if we make part of the public stable API, should we rename it to something less technical like `initialFlushDelay`?

Reply:
Yeah, as stated in a diff comment, this may not be a public option, but if it is, we will rename it!